File: create-dirs.py
import os

base_path = r'C:\Users\leona\Desktop\ProjetoDormeAqui\front-end\src\components\dashboard'
dirs = ['utils', 'hooks', 'atoms', 'sections', 'styles']

for dir_name in dirs:
    full_path = os.path.join(base_path, dir_name)
    os.makedirs(full_path, exist_ok=True)
    print(f'✓ Criado/Verificado: {full_path}')

print('\n✓ Todos os diretórios foram processados com sucesso!')

File: create-dirs.js
const fs = require('fs');
const path = require('path');

const basePath = 'C:\\Users\\leona\\Desktop\\ProjetoDormeAqui\\front-end\\src\\components\\dashboard';
const dirs = ['utils', 'hooks', 'atoms', 'sections', 'styles'];

dirs.forEach(dir => {
  const fullPath = path.join(basePath, dir);
  if (!fs.existsSync(fullPath)) {
    fs.mkdirSync(fullPath, { recursive: true });
    console.log(`✓ Criado: ${fullPath}`);
  } else {
    console.log(`✓ Já existe: ${fullPath}`);
  }
});

console.log('\n✓ Todos os diretórios foram processados com sucesso!');
